src/main/java/com/smartbear/postman/script/PostmanScriptParser.java

```diff
@@ -253,14 +254,12 @@ private void moreAssertions() {
     }
 
     private void nextToken() {
-        do {
-            tokenProcessed();
-            if (tokens.isEmpty()) {
-                lookahead = new Token(TokenType.END_OF_SCRIPT, "");
-            } else {
-                lookahead = tokens.getFirst();
-            }
-        } while (lookahead.getType() == TokenType.NEW_LINE);
```

Comment:
please describe why you removed this.

Reply:
To save the NEW_LINE to check it as a command separator later.